# ðŸš¨ to fix TypeError: esquery.parse is not a function in website
diff --git a/package.json b/package.json
index 3d818750d38a6d51207410ddd7352dd39ab035f2..da4286ab1dae709c8969fb6d412560ffb0f01bd0 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,6 @@
   "contributors": [],
   "description": "A query library for ECMAScript AST using a CSS selector like query language.",
   "main": "dist/esquery.min.js",
-  "module": "dist/esquery.esm.min.js",
   "files": [
     "dist/*.js",
     "dist/*.map",
